### PR TITLE
Empty script tags no longer cause the analyzer to fail

### PR DESCRIPTION
**Pull request by joineral32**
We were using iron-component-page and encountered a problem where our iron-component-page would be completely inoperable if a Polymer element contained an empty script tag. I tracked this issue down to Hydrolysis trying to access script.childNodes[0] regardless of whether or not script.childNodes was empty, and passing the resulting undefined into the jsParse method in the empty case.

### CODE REVIEW

**Comment by adewinter** [comment]
Nice!

File: lib/analyzer.js [hunk 1]
```diff
@@ -303,7 +303,7 @@ Analyzer.prototype._processScript = function _processScript(script, href) {
   var parsedJs;
   if (!src) {
     try {
-      parsedJs = jsParse(script.childNodes[0].value);
+      parsedJs = jsParse((script.childNodes.length) ? script.childNodes[0].value : '/* */');
```
Comment (garlicnation) [inline lib/analyzer.js]: Why an empty comment block instead of just the empty string?

**Comment by joineral32** [comment]
It is now using an empty string rather than an empty comment block.

**Comment by garlicnation** [comment]
If you squash this into one commit and remove `npm-debug.log`, I'm happy to merge it.

Thanks for the contribution!

**Comment by joineral32** [comment]
Squashed commits, rebased onto upstream and reran build. This should be all set. Thanks!

**Comment by garlicnation** [comment]
Merged in #154

**Pull request closed without merging**